.gitignore

```diff
@@ -0,0 +1,2 @@
+app/test.md
+
```

Comment:
- [ ] Kindly, add the `node_modules` directory in your .gitignore file to remove it from your repo because it could be added accidentally while setting up your project with Linters.

Reply:
Hi @RolandM99, thanks for your response. 
However, I have a question. 
I cannot see the **node_modules** in my repo. And I don't remember having it either. Could this be from linter?

Reply:
- [ ] Once some users run `npm install`, a directory named `node_modules` can be created in their local repository. This folder is commonly excluded from a remote repository because it has a large size and you shouldn’t add code that you didn’t write to the repository. To exclude it from your remote repository, you have to add it in your `.gitignore` file

Reply:
Ok. 
Thanks
Let me make the corrections.